internal/ycsb/ycsb.go

```diff
@@ -76,7 +93,14 @@ func Run(ctx context.Context, dir string, args []string) (*config.TestResults, e
 		},
 	}
 
-	if err := cmd.Run(); err != nil {
+	output, err := cmd.CombinedOutput()
```

Comment:
That breaks output streaming while the benchmark runs, making it hard to understand what is happening and when it will finish.

We should use some combination of https://pkg.go.dev/os/exec#Cmd.StdoutPipe, https://pkg.go.dev/io#TeeReader and/or https://pkg.go.dev/io#MultiWriter instead